### PR TITLE
Add x.com

### LINKED ISSUE #41: Twitter saving no longer works after transition to "x.com"
twitter.com now redirects to x.com on desktop (or at least this change has rolled out on my end), however, the addon doesn't recognize x.com as twitter, and does not activate.

### PR DESCRIPTION
**Pull request by geniuszxy**
Fix #41

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -95,7 +97,8 @@
 		},
 		{
 			"matches": [
-				"*://*.twitter.com/*"
+				"*://*.twitter.com/*",
+				"*://*.x.com/*"
```
Comment (Houdini111) [inline manifest.json:101]: Does this properly match images found even on the home feed? just `https://x.com`?

**Review by solorey — CHANGES_REQUESTED** [review]
According to MDN, [document.domain is deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Document/domain)
Instead of `window.document.domain` use `window.location.hostname`

File: manifest.json (hunk 1 above)
Comment (geniuszxy) [reply manifest.json:101]: Did you mean downloading images on https://x.com/home?

**Comment by Mirrorman95** [comment]
Brilliant! If I'm reading this code correctly, the pull request would allow ArtSaver to download images from X's new URL while still maintaining compatibility with URLs that use the site's old domain name.

**Comment by solorey** [comment]
Looks good to me.

**Pull request merged**